src/bin/miri.rs

```diff
@@ -573,6 +573,9 @@ fn main() {
             miri_config.mute_stdout_stderr = true;
         } else if arg == "-Zmiri-retag-fields" {
             miri_config.retag_fields = RetagFields::Yes;
+        } else if arg == "-Zmiri-fixed-schedule" {
+            miri_config.fixed_scheduling = true;
+            miri_config.preemption_rate = 0.0;
```

Comment:
We should just not auto-preempt when `fixed_scheduling` is set, so we don't have to set two fields here.